fastlane.gemspec

```diff
@@ -31,6 +31,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'addressable', '~> 2.3.8' # Support for URI templates
   spec.add_dependency 'xcode-install', '~> 1.0.1' # Download new Xcode versions
 
+  spec.add_dependency 'net/scp', '~> 1.2.1' # add SSH gem
```

Comment:
Please don't introduce new dependencies. Instead use the `Actions.verify_gem!('cocoapods')` method in your integration. Check out `cocoapods.rb` on how to do that :+1: